src/STW.ProcessingApi.Function/Constants/ChedType.cs

```diff
@@ -2,6 +2,16 @@ namespace STW.ProcessingApi.Function.Constants;
 
 public static class ChedType
 {
-    public const string Chedpp = "CHED-PP";
-    public const string Chedp = "CHED-P";
+    public const string Cheda = "CHEDA";
+    public const string Chedp = "CHEDP";
```

Comment:
Why did you change CHED-PP and CHED-P to CHEDPP and CHEDP? Both the schema we were working from and the notification schema have these original values.